plugins/sentry.client.ts

```diff
@@ -0,0 +1,33 @@
+import * as Sentry from '@sentry/vue'
+
+export default defineNuxtPlugin((nuxtApp) => {
+  const router = useRouter()
+  const { public: { sentry } } = useRuntimeConfig()
+
+  if (!sentry.dsn) {
+    return
+  }
+
+  Sentry.init({
+    app: nuxtApp.vueApp,
+    dsn: sentry.dsn,
+    environment: sentry.environment,
+    integrations: [
+      Sentry.browserTracingIntegration({ router }),
+      Sentry.replayIntegration(),
+    ],
+
+    // Set tracesSampleRate to 1.0 to capture 100%
+    // of transactions for performance monitoring.
+    // We recommend adjusting this value in production
+    tracesSampleRate: 1.0, // Change in prod
```

Comment:
We are in prod.

```suggestion
    tracesSampleRate: 0.05,
```

Reply:
You mean 0.5 (50% of errors sent) no ?

Reply:
It is for  "performance monitoring" not errors.